controllers/storagecluster/cephconfig.go

```diff
@@ -162,5 +161,28 @@ func getRookCephConfig(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) (s
 			return "", fmt.Errorf("failed to set network configuration for rook: %v", err)
 		}
 	}
+	// configure rbd mirroring debug features by default if mirroring is enabled
+	if sc.Spec.Mirroring.Enabled {
+		var err error
+		// [global] section
+		rookConfigData, err = updateRookConfig(rookConfigData, globalSectionKey, "rbd_mirror_die_after_seconds", "3600")
+		if err != nil {
+			return "", fmt.Errorf("failed to set rbd mirroring debug features for rook config: %v", err)
+		}
+
+		rbd_mirror_debug :=
+			`[client.rbd-mirror.a]
+debug_ms = 1
+debug_rbd = 20
+debug_rbd_mirror = 30
+log_file = /var/log/ceph/$cluster-$name.log
+[client.rbd-mirror-peer]
+debug_ms = 1
+debug_rbd = 20
+debug_rbd_mirror = 30
+log_file = /var/log/ceph/$cluster-$name.log
```

Comment:
Doing it this way doesn't cover `mgr/rbd_support/log_level` and also means that none of these settings can be easily reverted (we have learned that the hard way with `rbd_mirror_die_after_seconds`).  I would prefer if all of these were set with `ceph config set` so that users can easily disable verbose logging if it ends up hurting them.